src/main/java/org/prebid/server/proto/openrtb/ext/request/rtbhouse/ExtImpRtbhouse.java

```diff
@@ -0,0 +1,27 @@
+package org.prebid.server.proto.openrtb.ext.request.rtbhouse;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Value;
+
+import java.math.BigDecimal;
+
+/**
+* Defines the contract for bidrequest.imp[i].ext.rtbhouse
+*/
```

Comment:
We tend to avoid unnecessery comments, please, remove.

Reply:
removed